db/migrations/0015.add_mivideo_creation_tables.py

```diff
@@ -0,0 +1,28 @@
+'''
+Migration for MiVideo media started hourly table
+'''
+
+from yoyo import step
+
+__depends__ = {'0012.add_mivideo_usage_table'}
+
+
+steps = [
+    step('''
+        CREATE TABLE IF NOT EXISTS mivideo_media_created (
```

Comment:
The intention for the table was to store data about when media was created and with which Canvas course is it related.  There are several other attributes of the media available, but we're not capturing them here.  If we want to be really strict here, we could:

1. Keep this name and remove the `duration` column, because that's not strictly related to the creation, identity, or LMS relationship of the media.
1. Change the name to `mivideo_media` and add all the various attributes of the media that could possibly be of use to us now or in the near future.

Probably not surprisingly, I don't think we need to change anything here.

Reply:
I think the answer to this will depend on what the business question is that we are trying to answer. I think I'm in favor of `mivideo_media` if you keep `duration`; I don't know whether that element is needed or not.